src/main/java/org/folio/inventory/validation/experimental/InProcessTargetValidator.java

```diff
@@ -0,0 +1,23 @@
+package org.folio.inventory.validation.experimental;
+
+import org.folio.inventory.domain.items.ItemStatusName;
+
+import static java.util.Set.of;
+
+public class InProcessTargetValidator extends AbstractTargetValidator {
+
+  public InProcessTargetValidator() {
+    super(ItemStatusName.IN_PROCESS, of(
+      ItemStatusName.AVAILABLE,
+      ItemStatusName.AWAITING_DELIVERY,
+      ItemStatusName.AWAITING_PICKUP,
+      ItemStatusName.IN_TRANSIT,
+      ItemStatusName.LOST_AND_PAID,
+      ItemStatusName.MISSING,
+      ItemStatusName.ORDER_CLOSED,
+      ItemStatusName.PAGED,
+      ItemStatusName.WITHDRAWN
+    ));
+  }
+
```

Comment:
Please remove this redundant blank line, I don't think it improves the readability of the code